src/tests/ie_test_utils/common_test_utils/file_utils.cpp

```diff
@@ -43,8 +49,74 @@ std::string getExecutableDirectory() {
     return ov::util::get_directory(path);
 }
 
-std::string getModelFromTestModelZoo(const std::string & relModelPath) {
+std::string getCurrentWorkingDir() {
+    std::string path;
+#ifdef _WIN32
+    char * buffer = _getcwd(NULL, 0);
+    if (buffer != NULL) {
+        path = std::string(buffer);
+        free(buffer);
+    }
+#else
+    char buffer[PATH_MAX];
+    auto result = getcwd(buffer, sizeof(buffer));
+    if (result != NULL) {
+        path = std::string(buffer);
+    } else {
+        int error = errno;
+        std::ostringstream str;
+        str << "Can't get access to the current working directory, error:" << error;
+        throw std::runtime_error(str.str());
+    }
+#endif
+    return path;
+}
+
+std::string getModelFromTestModelZoo(const std::string& relModelPath) {
     return ov::util::path_join({CommonTestUtils::getExecutableDirectory(), relModelPath});
 }
 
-} // namespace CommonTestUtils
+std::string getRelativePath(const std::string& from, const std::string& to) {
+    auto split_path = [](const std::string& path) -> std::vector<std::string> {
+        std::string sep{ov::util::FileTraits<char>::file_separator};
+        std::vector<std::string> retvalue;
+        size_t start = 0;
+        size_t end = 0;
+        std::string token;
+        while ((end = path.find(sep, start)) != std::string::npos) {
+            token = path.substr(start, end - start);
+            start = end + 1;
+            retvalue.push_back(token);
+        }
+        retvalue.push_back(path.substr(start));
+        return retvalue;
+    };
+
+    auto from_vec = split_path(from);
+    auto to_vec = split_path(to);
+
+    auto mismatch_it = std::mismatch(from_vec.begin(), from_vec.end(), to_vec.begin());
+    if (mismatch_it.first == from_vec.end() && mismatch_it.second == to_vec.end()) {
+        return {};
+    }
+
+    std::string separator(1, ov::util::FileTraits<char>::file_separator);
+    std::string output;
+    if (mismatch_it.first != from_vec.end()) {
```

Comment:
Could we add comments to this methods?
Does it work in case: `/aaa/bbb/cc/test_app` and cwd `/aaa/bbb/ddd/eee/`?

Reply:
Yes, it works the generated path is: `../../../cc/test_app`

Reply:
@artkuli could you also check case with `/` and `/some/path`?

Reply:
@vurusovs When start path is: `/` and destination path is: `/test/test_app` it generates: `../test/test_app` 
In the case where start path is: `/test/test_app` and destination is: `/` generates: `../../`

Reply:
Case with `../test/test_app` isn't correct, but it works, so I have no issues here

Reply:
But about the first case:
> /aaa/bbb/cc/test_app and cwd /aaa/bbb/ddd/eee/
It generates wrong output `../../../cc/test_app`, it should be `../../cc/test_app` (2 times of `../`)

Looks like there is a problem with `/path/to` and `/path/to/` difference (with last symbol which is /)

Reply:
@vurusovs  Good catch! Thanks! I figured it out, now the path is correct: `../../cc/test_app` also in the case where start path is: `/` and destination path is: `/test/test_app`, now  it generates: `test/test_app`